indico/modules/events/reminders/templates/emails/event_reminder.txt

```diff
@@ -23,6 +23,14 @@ It will take place at {{ render_location() }}
 You can access the full event here:
 {{ url }}
 
+{%- if with_description and event.description %}
+
+{% filter underline %}Description{% endfilter %}
+
+{{ event.description| html2text }}
```

Comment:
missing space before `|`

Reply:
Adding ` | trim` after converting to text would probably be a good idea as well

Reply:
done